shell/edit/provisioning.cattle.io.cluster/tabs/Etcd.vue

```diff
@@ -3,7 +3,7 @@ import { LabeledInput } from '@components/Form/LabeledInput';
 
 import { RadioGroup } from '@components/Form/Radio';
 
-import S3Config from './S3Config';
+import S3Config from '@shell/edit/provisioning.cattle.io.cluster/S3Config';
```

Comment:
this is only used from this component. to keep things neat in the prov cluster folder can the same be done for this?

provisioning.cattle.io.cluster/tabs/etcd/index.vue
provisioning.cattle.io.cluster/tabs/etcd/S3Config.vue